boards/atxmega-a1u-xpro/Makefile.dep

```diff
@@ -1,5 +1,10 @@
 USEMODULE += boards_common_atxmega
 
+ifneq (,$(filter netdev_default,$(USEMODULE)))
+  USEMODULE += at86rf231
+  USEMODULE += periph_cpuid
+endif
+
```

Comment:
Note: This is only intended for transceivers that are inherently part of the board.

If the AT86RF231 breakout board is advertised as being plug-and-pray compatible with the xmega-a1u-xpro, I agree that it does make sense to provide a configuration for this transceiver. But uses still need to select non-default hardware by hand.

Using `periph_cpuid` should by the way not be necessary. If it is, you found a bug in the dependency modeling. (luid should have `FEAUTRES_OPTIONAL += periph_cpuid` added. And used features with names matching `periph_%` will automatically pull in a module with the corresponding name.

Update: OK, I see that you implemented a custom EUI provided based on the CPU-ID. But in that case, the use of the EUI provider should pull in `periph_cpuid`, but using `FEAUTRES_REQUIRED += periph_cpuid`.

Reply:
Let's keep simple. I'll add only the driver and header configuration for now.